__tests__/app.test.js

```diff
@@ -430,4 +430,36 @@ describe("Votes", () => {
             })
         })
     })
+})
+describe("Users", () => {
+    describe("GET Users", () => {
+        test("GET: 200 - responds with an array of all users", () => {
+            return request(app).get("/api/users")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.users).toHaveLength(4)
+            })
+        })
+        test("GET: 200 - the array of users has the required property keys with correct value types", () => {
+            return request(app).get("/api/users")
+            .expect(200)
+            .then(({body}) => {
+                const users = body.users
+                const keys = ["username", "name", "avatar_url"]
+                users.forEach(user => {
+                    expect(Object.keys(user)).toEqual(keys)
+                    expect(typeof user.username).toBe("string")
+                    expect(typeof user.name).toBe("string")
+                    expect(typeof user.avatar_url).toBe("string")
```

Comment:
to level up these assertions, consider using some of the more advanced jest matchers such as `.toHaveProperty` with `expect.any()` as the second argument. These matchers are more semantic / readable 😄